src/Shared/DashboardConfigNames.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting;
+
+internal static class DashboardConfigNames
+{
+    internal const string DashboardConfigFilePathVariableName = "DOTNET_DASHBOARD_CONFIG_FILE_PATH";
+    internal const string DashboardInsecureAllowAnonymousVariableName = "DOTNET_DASHBOARD_INSECURE_ALLOW_ANONYMOUS";
+    internal const string DashboardOtlpAuthModeVariableName = "Dashboard:Otlp:AuthMode";
+    internal const string DashboardOtlpPrimaryApiKeyVariableName = "Dashboard:Otlp:PrimaryApiKey";
+    internal const string DashboardOtlpSecondaryApiKeyVariableName = "Dashboard:Otlp:SecondaryApiKey";
```

Comment:
These are environment variables for the config system right? This won't work on anything but windows.

```suggestion
    internal const string DashboardOtlpAuthModeVariableName = "Dashboard__Otlp__AuthMode";
    internal const string DashboardOtlpPrimaryApiKeyVariableName = "Dashboard__Otlp__PrimaryApiKey";
    internal const string DashboardOtlpSecondaryApiKeyVariableName = "Dashboard__Otlp__SecondaryApiKey";
```

Reply:
These keys are used to both set env vars and read from config.

Will `config["Dashboard__Otlp__AuthMode"]` work or must it be `config["Dashboard:Otlp:AuthMode"]`?

If `config["Dashboard__Otlp__AuthMode"]` doesn't work then there will just need to be a couple of extra constants for the env var names.

Reply:
You need separate ones for env variables vs config then.